### PR TITLE
Update to the /core page

### LINKED ISSUE #4508: /core 18 - build
Build the new page per the wireframe and design

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Complete update to the /core page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/core](http://0.0.0.0:8001/core)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1G-h1eWiBSKqG9oWB15Pup8VDe1axBUwQGvtkwJ9g8T0/edit#heading=h.xnpkx4ypduk)

## Issue / Card

Fixes #4508

### CODE REVIEW

File: templates/core/index.html [hunk 1]
```diff
@@ -5,194 +5,433 @@
...
+    <li class="prefix-4 col-6 mobile-col-6 tablet-col-6"><img class="" src="{{ ASSET_SERVER_URL }}94ea495b-2018-logo-Intel.svg" alt="Intel logo" /></li>
+    <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}b1767848-2018-logo-raspberry-pi.svg" alt="Raspberry PI logo" /></li>
+  </ul>
+  <ul class="row">
+    <li class="prefix-4 col-6 mobile-col-6 tablet-col-6"><img src="{{ ASSET_SERVER_URL }}0970d532-2018-logo-dell.svg" alt="Dell logo" /></li>
+    <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}45fd87b5-2018-logo-rigado.svg" alt="Rigado logo" /></li>
+  </ul>
+  <div class="row">
+    <div class="col-12 u-align--center">
+      <p class="u-no-max-width">Canonical and partners speed your device development.</p>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <p class="u-no-max-width">Canonical and partners speed up your device development.</p>
```

File: templates/core/index.html [hunk 2]
```diff
@@ -5,194 +5,433 @@
...
+      <li class='p-matrix__item'>&nbsp;</li>
+    </ul>
+  </div>
+</section>
+
+<section class='p-strip--image p-banner-core'>
+  <div class='row u-vertically-center'>
+    <div class='col-8'>
+      <h2>Reduce the cost of Linux</h2>
+      <h4>Use Ubuntu Core free. Pay only for extra services.</h4>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <h4>Use Ubuntu Core for free. Pay only for extra services.</h4>
```

File: templates/core/index.html [hunk 3]
```diff
@@ -5,194 +5,433 @@
...
 
 {% block content %}
-<div class="p-strip--light is-deep is-bordered">
-  <div class="row">
-    <div class="col-6">
-      <h1>Ubuntu Core</h1>
+<div class="p-strip--image p-banner-core-hero is-deep is-dark is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h1>Make secure devices with Ubuntu Core</h1>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <h1>Make secure IoT devices with Ubuntu Core</h1>
```

File: templates/core/index.html [hunk 4]
```diff
@@ -1,199 +1,581 @@
...
 
 {% block content %}
-<div class="p-strip--light is-deep is-bordered">
-  <div class="row">
-    <div class="col-6">
-      <h1>Ubuntu Core</h1>
+<section class="p-strip--image p-banner-core-hero is-deep is-dark">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h1>Make secure IoT devices with Ubuntu Core</h1>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <h1>Build secure IoT devices with Ubuntu Core</h1>
```

File: templates/core/index.html [hunk 5]
```diff
@@ -1,199 +1,581 @@
...
-      <h4>Secure by default</h4>
-      <p>Automatic updates ensure that critical security issues are addressed in the field, even if a device is unattended.</p>
+    <hr class="p-separator" />
+  </div>
+
+  <div class="row">
+    <div class="col-6">
+      <h4>10 years security updates, by Canonical</h4>
+      <p class='p-heading--five'>That's a decade of sleeping soundly.</p>
+      <p>Ubuntu Core 18 gets 10 years of Canonical maintenance from Ubuntu 18.04 LTS. Your smallest devices are now just as well covered as your servers.</p>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <p>Ubuntu Core 18 gets 10 years of Canonical maintenance from Ubuntu 18.04 LTS. Your smallest devices are now as secure as your servers.</p>
```

File: templates/core/index.html [hunk 6]
```diff
@@ -1,199 +1,581 @@
...
+</section>
+
+<section class='p-strip'>
   <div class="row">
-    <div class="col-12  u-align--center">
-      <img src="{{ ASSET_SERVER_URL }}79334500-Ubuntu_Classic_vs_Core_Desktop.svg" alt="" />
+    <div class="col-6">
+      <h4>Manufacturer’s update control</h4>
+      <p class='p-heading--five'>Decide which updates go to your devices.</p>
+      <p>As a device manufacturer or a snap publisher, you decide which updates are certified and delivered to your devices. Digital signatures certify compatibility for precise update control.</p>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <p>As a device manufacturer or a snap publisher, you decide which updates are signed, certified and delivered to your devices. </p>
```

File: templates/core/index.html [hunk 7]
```diff
@@ -1,199 +1,581 @@
...
+      <img src="{{ ASSET_SERVER_URL }}08af835e-Snap+Store+shopping+icon.svg" width="200" alt="">
     </div>
-    <div class="col-6 u-align--center">
-      <img src="{{ ASSET_SERVER_URL }}c6a45f9c-Ubuntu_Classic_vs_Core_-mobile2.svg" alt="" width="260" style="margin-top: 1rem;"/>
+    <div class="col-6">
+      <h4>App store included</h4>
+      <p class='p-heading--five'>Your own app store. Your own software portfolio.</p>
+      <p>Every device built with Ubuntu Core has a secure app store. Use the global
+store for access to all the standard apps, or curate your own collection.</p>
+      <p><a class="p-button--neutral is-inline" href="/internet-of-things/appstore" style="margin-left: 0;"><span class="p-link--external">See what an app store can do for you</span></a></p>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <p><a class="p-button--neutral is-inline" href="/internet-of-things/appstore" style="margin-left: 0;"><span class="p-link--external">See what an app store can do for your things</span></a></p>
```

File: templates/core/index.html [hunk 8]
```diff
@@ -1,199 +1,581 @@
...
+  </div>
+
+  <div class="row u-equal-height">
     <div class="col-6">
-      <h2>The smallest Ubuntu</h2>
-      <p>Ubuntu Core is smaller than competing &ldquo;micro&rdquo; container OS offerings. It is small because it&rsquo;s really just a base filesystem. Apps are delivered as snaps, alongside a free choice of container runtimes and coordination systems. And because it&rsquo;s got a smaller attack surface, it&rsquo;s much more secure.</p>
+      <h4>Enterprise management</h4>
+      <p class='p-heading--five'>Absolute control over every device in the building.</p>
+      <p>Enterprises gain complete audit and control over every piece of software on every single device on the network. Regardless of manufacturer.</p>
+      <p>You know exactly which kernel version and which OS version is running. You decide when updates happen. You decide which versions, too.</p>
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
      <p>Know exactly which kernel version and which OS version is running. You decide when updates happen. You decide which versions, too.</p>
```

**Comment by anasereijo** [comment]
@anthonydillon 

- [ ] insert [this image](https://assets.ubuntu.com/v1/de23ae0e-logos.svg) on "Choice of ARM or x86" section to look like this:
logos

**Comment by anthonydillon** [comment]
@Greggless @anasereijo done

**Comment by anasereijo** [comment]
@anthonydillon thanks for the changes but the sticky nav looks the same?

**Comment by anasereijo** [comment]
Everything looks tidy and nice, just waiting for the illustrations to be updated and then I can +1.

(Would be nice to fix the sticky nav issue on small screens but, as discussed with @anthonydillon, that might take a bit of time and work so maybe we can try to fix it other time)

**Comment by Greggless** [comment]
The 'strict confinement' illustration uses `#333333`

plz update, plz

File: templates/core/index.html [hunk 9]
```diff
@@ -1,199 +1,585 @@
...
-      </div>
-      <div class="col-6 u-vertically-center u-align--center">
-        <img src="{{ ASSET_SERVER_URL }}5051e6e5-core_logo.svg" alt="" width="300" />
-      </div>
+    <hr class="p-separator" />
+  </div>
+
+  <div class="row">
+    <div class="col-6 u-hide--small u-align--center">
+      <img src="{{ ASSET_SERVER_URL }}d7eb85bc-1B-confinement.svg" width="350" alt="">
```
Comment (Greggless) [inline templates/core/index.html]:
```suggestion
       <img src="{{ ASSET_SERVER_URL }}256d7d18-1B-confinement+.svg" width="350" alt="">
```

Comment (anthonydillon) [reply templates/core/index.html]: done

**Pull request merged**